### PR TITLE
Update spec for python 3.11 build

### PR DESCRIPTION
**Pull request by smarlowucf**
Update vcrpy cassettes for 3.11

### CODE REVIEW

File: python3-aws-regions.spec [hunk 1]
```diff
@@ -15,43 +15,42 @@
...
 Version:        0.1.2
 Release:        0
 Summary:        API for retrieving the most up-to-date AWS region list
 License:        MIT
 URL:            https://github.com/SUSE-Enceladus/aws-regions
 Source:         https://files.pythonhosted.org/packages/source/a/aws-regions/aws-regions-%{version}.tar.gz
 BuildRequires:  python-rpm-macros
-BuildRequires:  python3-requests
-BuildRequires:  python3-pytest
-BuildRequires:  python3-vcrpy
```
Comment (apozsuse) [inline python3-aws-regions.spec:29]: vcr  in `BuildRequires` is missing, but it's still imported in `tests\test_endpoints.py`. Is this intended?

Comment (smarlowucf) [reply python3-aws-regions.spec:29]: Not intended. Was testing prior to vcrpy being packaged so dropped it from the spec.

Comment (smarlowucf) [reply python3-aws-regions.spec:29]: Fixed and rebased

**Comment by smarlowucf** [comment]
Going ahead with merge to start packaging. Please still provide any feedback and it can be updated in another pr.

**Comment by rjschwei** [comment]
LGTM

**Pull request merged**